www/.htaccess

```diff
@@ -14,13 +14,13 @@ RewriteRule ^(doc/) http://docs.scipy.org/ [R=permanent]
 RewriteRule ^(Download|Installing_SciPy) /install.html [R=permanent]
 RewriteRule ^(scipy_Example_List) https://docs.scipy.org/doc/scipy/reference/ [R=permanent]
 RewriteRule ^(Numpy_Example_List) https://docs.scipy.org/doc/numpy/reference/routines.html [R=permanent]
```

Comment:
This is rather strange, as this link is the same as it was below before replacement and seems to work OK. What is the difference?

Reply:
@charris quoting @mattip from another thread where I asked a similar question (I think):

> NumPy is intending to host its own documentation going forward at numpy.org/doc